### PR TITLE
adding view for number of likes and tests

### PR DESCRIPTION
**Pull request by marinaRomanchuk**

### CODE REVIEW

File: posts/views.py [hunk 1]
```diff
@@ -39,6 +39,11 @@ def list(self, request) -> Response:
 class LikeDislikeViewSet(viewsets.ModelViewSet):
     permission_classes = (IsAuthenticated,)
 
+    def get(self, request, pk: int):
+        is_like = "dislike" not in request.path
+        count = LikeDislike.objects.filter(post_id=pk, is_like=is_like).count()
+        return Response({"count": count}, status=status.HTTP_200_OK)
```
Comment (revimi) [inline posts/views.py]: We can rather have a view that returns both numbers of likes and dislikes. It will help to decrease client app calls count

**Review by revimi — CHANGES_REQUESTED** [review]

File: posts/views.py (hunk 1 above)
Comment (marinaRomanchuk) [reply posts/views.py]: Done

File: posts/views.py [hunk 2]
```diff
@@ -39,6 +39,15 @@ def list(self, request) -> Response:
 class LikeDislikeViewSet(viewsets.ModelViewSet):
     permission_classes = (IsAuthenticated,)
 
+    def get(self, request, pk: int):
```
Comment (revimi) [inline posts/views.py:42]: What is the URL for this endpoint?

Comment (marinaRomanchuk) [reply posts/views.py:42]: both http://127.0.0.1:8000/api/posts/1/like/ and http://127.0.0.1:8000/api/posts/1/dislike/ for post 1

Comment (revimi) [reply posts/views.py:42]: Let's have something like
`/api/posts/{id}/likes_count/` or `/api/posts/{id}/likes/count/` or `/api/posts/{id}/likes/stats/` or `/api/posts/{id}/likes/`

Comment (marinaRomanchuk) [reply posts/views.py:42]: Done, now it's `/api/posts/{id}/likes/count/`

**Review by revimi — APPROVED** [review]

**Pull request merged**